### DISCUSSION TITLE
Integrating Custom URDF Models in pinocchio

### DISCUSSION THREAD

**Discussion by fukangl** [stack-of-tasks/pinocchio#2107] [Q&A] [answered]
Hi, I am currently utilizing Crocoddyl for my research project and need assistance with loading a custom URDF model,  which isn't included in the pinocchio repository. Could you offer some guidance on how to proceed with this?

**Comment by jcarpent** [comment, marked as answer]
Please see this [URDF loading example](https://github.com/stack-of-tasks/pinocchio/blob/97a00a983e66fc0a58667f9671e2d806ba9b730b/examples/overview-urdf.py).
